website/client/components/userMenu/profileStats.vue

```diff
@@ -120,7 +120,7 @@
       .row.title-row
         .col-12.col-md-6
           h3(v-if='userLevel100Plus', v-once, v-html="$t('noMoreAllocate')")
-          h3(v-if='user.stats.points || userLevel100Plus')
```

Comment:
looks good, why was this check removed?

Reply:
The title was hidden when the user spent points, but that looked odd. Must have been pulled over from the old site.